srcs/color.c

```diff
@@ -5,11 +5,6 @@ int	get_trgb(int t, int r, int g, int b)
 	return (t << 24 | r << 16 | g << 8 | b);
 }
 
```

Comment:
why did you remove get_t ?

Reply:
We're not using it at all because no objects are transparent. I doubt Minilibx can even handle transparency actually?
Also this way this file has max 5 functions!

Reply:
the mac version of minilibx can definetely handle transparency, which is why everything was transparent on your mac at the beginning :p maybe the linux version of minilibx can't, that I don't know. the limit of functions per file alone shouldn't be a strong enough argument, we can just create another file

Reply:
Fair enough on the limit of functions/minilibx, but why let a function in which is never called?

Reply:
It was being called 😂 It's as useful/useless (that's discussable I agree) as the other 3 functions (get_r/g/b)

Reply:
I mean only in the print.. but ok added it back!